### PR TITLE
credential

### PR DESCRIPTION
**Pull request by AlexTurning**

### CODE REVIEW

**Comment by shubham-padia** [comment]
Please resolve the merge conflicts.

File: index.html [hunk 1]
```diff
@@ -213,8 +234,31 @@ <h2 style="color: black;"><span>PiRong</span> Koh</h2>
...
+                                       <a href="#"
+                                          target="_blank">
+                                           <i class="fa fa-twitter" aria-hidden="false"></i></a>
+                                   </p>
+                               </figcaption>
+                           </figure>
+                       </div>
+                 </div>
+              </div>
             </div>
```
Comment (jigyasa-grover) [inline index.html:262]: @AlexTurning May I know why have you added twice ?

**Comment by jigyasa-grover** [comment]
@AlexTurning 
* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students section"
* Resolve merge conflicts, this [How To Manually Fix Git Merge Conflicts (video)](https://www.youtube.com/watch?v=g8BRcB9NLp4) might be helpful.

**Review by daminisatya — CHANGES_REQUESTED** [review]
Requesting changes mentioned by the mentors! Thank you

**Comment by abishekvashok** [comment]
Please don't update index.htmk to add your blog just add it at blogs.yml only and place the picture. :) Happy coding 😉

**Comment by jigyasa-grover** [comment]
Wrt [rules in CONTRIBUTING.md](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md), closing this.

**Pull request closed without merging**